CAIPs/caip-275.md

```diff
@@ -350,9 +337,9 @@ export class ENS implements NameResolver {
 
 # Rationale
```

Comment:
it might be good to add a kind of high-level rationale, like "this doesn't compete with other authentication method, it is just an opt-in discovery system that name-controllers can use and that applications can trust to automate one friction point."
it might also be the right section to explain how future authentication methods (#teamWearables) get added to the mix-- followup CAIPs? are you providing a template for those? 😄